javascript/example_code/iam/iam_deleteaccountalias.js

```diff
@@ -22,8 +22,8 @@ var iam = new AWS.IAM({apiVersion: '2010-05-08'});
 
 iam.deleteAccountAlias({AccountAlias: process.argv[2]}, function(err, data) {
   if (err) {
-    console.log("Error", err);
+    throw err;
   } else {
-    console.log("Success", data);
+    console.log('Account alias ' + process.argv[2] + ' deleted.');
```

Comment:
I updated many of the success messages, because most of these operations don't return data if they succeed. This meant many of the operations would print:
`Success null` if the script ran successfully.